lms/djangoapps/teams/models.py

```diff
@@ -88,6 +91,14 @@ class Meta(object):
     user = models.ForeignKey(User)
     team = models.ForeignKey(CourseTeam, related_name='membership')
     date_joined = models.DateTimeField(auto_now_add=True)
+    last_activity_at = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
+        """ Customize save method to set the last_activity_at if it does not currently exist. """
+        if not self.last_activity_at:
```

Comment:
@dianakhuang I'd like to talk to you about this bit of code-- I don't see any reason to do it, as we will still have to handle None for memberships that have never been saved at the migration (which will be all of them I think... I don't know why an existing membership would be saved again).